protocol/messenger_communities.go

```diff
@@ -1308,6 +1308,19 @@ func (m *Messenger) CancelRequestToJoinCommunity(request *requests.CancelRequest
 		return nil, err
 	}
 
+	if !community.AcceptRequestToJoinAutomatically() {
+		// send cancelation to community admins also
+		rawMessage.Payload = payload
+
+		privilegedMembers := community.GetPrivilegedMembers()
+		for _, privilegedMember := range privilegedMembers {
+			_, err := m.sender.SendPrivate(context.Background(), privilegedMember, &rawMessage)
```

Comment:
I can add it, but I don't think there is a scenario where you can send it to yourself, since if you cancel a request to join, that means you're not part of the community, hence not a privileged user.